src/misc.c

```diff
@@ -222,10 +222,14 @@ static int luv_getrusage(lua_State* L) {
 }
 
 static int luv_cpu_info(lua_State* L) {
-  uv_cpu_info_t* cpu_infos;
-  int count, i;
+  uv_cpu_info_t* cpu_infos = NULL;
+  int count = 0, i;
   int ret = uv_cpu_info(&cpu_infos, &count);
-  if (ret < 0) return luv_error(L, ret);
+  if (ret != 0)
```

Comment:
Why did this change to `!= 0`? Libuv errors are always negative: http://docs.libuv.org/en/v1.x/errors.html

Reply:
Revert that